include/onnxruntime/core/framework/data_types.h

```diff
@@ -141,6 +141,10 @@ class DataTypeImpl {
     return type_ == GeneralType::kNonTensor;
   }
 
+  bool IsInvalidType() const {
+    return type_ == GeneralType::kInvalid;
+  }
```

Comment:
It is not clear how type can be invalid. What does it mean in practice?

Reply:
I can remove it. I need to walk through the code to see what it means.